packages/bull-vault/test/e2e/BullStrategyTestFork.t.sol

```diff
@@ -96,6 +96,32 @@ contract BullStrategyTestFork is Test {
         assertEq(IERC20(usdc).balanceOf(user1), usdcToBorrow);
     }
 
+    function testSecondDeposit() public {
+        uint256 crabToDepositInitially = 10e18;
+
+        vm.startPrank(user1);
+        (uint256 wethToLend, uint256 usdcToBorrow) = _deposit(crabToDepositInitially);
+        vm.stopPrank();
+
+        assertEq(bullStrategy.balanceOf(user1), crabToDepositInitially);
+        assertEq(IEulerDToken(dToken).balanceOf(address(bullStrategy)), usdcToBorrow);
+        assertTrue(wethToLend.sub(IEulerEToken(eToken).balanceOfUnderlying(address(bullStrategy))) <= 1);
```

Comment:
This is fine for this test, but if you add a fuzzing one, note that sometimes it gets to 2 wei difference.